api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.6.x.xml

```diff
@@ -133,5 +133,17 @@
 		<comment>Adding not null constraint to encounter_diagnosis.dx_rank. TRUNK-6081</comment>
 		<addNotNullConstraint tableName="encounter_diagnosis" columnName="dx_rank" columnDataType="int"/>
 	</changeSet>
-	
+
+	<changeSet id="TRUNK-6189-2023-09-06" author="mherman22">
```

Comment:
I'd still avoid doing this at the database level...

Reply:
Not getting you. So the requirement to make the name unique is no more,right?.

Reply:
No, we want to make sure that the name is unique among non-retired provider attribute types. However, we don't normally do this via unique indexes on the database; we usually just work with the validator to do so, so this DB-level rule is unnecessary, but we should validate the uniqueness of the name in the validator.